prisma/migrations/20240830130414_optional_company_profile_summary/migration.sql

```diff
@@ -1,3 +1,3 @@
--- AlterTable
+-- AlterTable e
```

Comment:
typo

Reply:
Resolved in b6d94d5a8fa385c07f09aa2db7c14964fd72d34d